app/Models/User.php

```diff
@@ -68,4 +68,8 @@ public function sendPasswordResetNotification($token) {
 		}
 	}
 
+	public function routeNotificationForNexmo($notification) {
+		return $this->user->data['phone_number'];
```

Comment:
this needs to be a search for field similar to "phone_number" and not just phone number